intg/src/main/java/org/apache/atlas/AtlasErrorCode.java

```diff
@@ -274,6 +274,8 @@ public enum AtlasErrorCode {
     RUNTIME_EXCEPTION(500, "ATLAS-500-00-020", "Runtime exception {0}"),
     KEYCLOAK_INIT_FAILED(500, "ATLAS-500-00-022", "Failed to initialize keycloak client: {0}"),
 
+    MAINTENANCE_MODE_ENABLED(503, "ATLAS-503-00-001", "Atlas is in maintenance mode for this specific operation. Please try again later."),
```

Comment:
Even if we don't fix it, we should know what behaviour would user observe on the product when the do CUD operations during maintenance window (do they see some visible error, or calls fail silently, is error legible to user, etc.).

Reply:
<img width="587" alt="image" src="https://github.com/atlanhq/atlas-metastore/assets/59254445/0fa9c3a9-a70e-495f-8a1d-a2957f3f3395">
They see this error message